### PR TITLE
ginkgo: Get rid of K8sUpdates

### PR DESCRIPTION
**Pull request by brb**
It got replaced by ci-e2e-upgrade.

### CODE REVIEW

File: .github/actions/ginkgo/main-focus.yaml [hunk 1]
```diff
@@ -189,7 +189,7 @@ include:
     cliFocus: 'K8sDatapathServicesTest Checks E/W loadbalancing \\(ClusterIP, NodePort from inside cluster, etc\\) TFTP|K8sDatapathServicesTest Checks E/W loadbalancing \\(ClusterIP, NodePort from inside cluster, etc\\) with'
 
   ###
-  # K8sDatapathServicesTest Checks E/W loadbalancing (ClusterIP, NodePort from inside cluster, etc) Tests NodePort inside cluster (kube-proxy)
```
Comment (aanm) [inline .github/actions/ginkgo/main-focus.yaml]: drop this change

Comment (brb) [reply .github/actions/ginkgo/main-focus.yaml]: I can't - my editor does it.

File: test/k8s/updates.go [hunk 2]
```diff
@@ -1,536 +0,0 @@
...
-				helpers.CiliumStableVersion, helpers.GetCurrentK8SEnv()))
-			return
-		}
-
-		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
-
-		demoPath = helpers.ManifestGet(kubectl.BasePath(), "demo.yaml")
-		l7Policy = helpers.ManifestGet(kubectl.BasePath(), "l7-policy.yaml")
-		migrateSVCClient = helpers.ManifestGet(kubectl.BasePath(), "migrate-svc-client.yaml")
-		migrateSVCServer = helpers.ManifestGet(kubectl.BasePath(), "migrate-svc-server.yaml")
```
Comment (tklauser) [inline test/k8s/updates.go:79]: These manifests look unused now. Delete them as well?

Comment (brb) [reply test/k8s/updates.go:79]: I saw folks referencing them in other repos :grimacing:

File: test/k8s/updates.go [hunk 3]
```diff
@@ -1,536 +0,0 @@
...
-			Skip(fmt.Sprintf(
-				"Cilium %q is not supported in K8s %q. Skipping upgrade/downgrade tests.",
-				helpers.CiliumStableVersion, helpers.GetCurrentK8SEnv()))
-			return
-		}
-
-		kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
-
-		demoPath = helpers.ManifestGet(kubectl.BasePath(), "demo.yaml")
-		l7Policy = helpers.ManifestGet(kubectl.BasePath(), "l7-policy.yaml")
```
Comment (tklauser) [inline test/k8s/updates.go:77]: l7-policy.yaml looks unused as well, but https://github.com/cilium/cilium/pull/29430/files#r1407527644 might apply too? :disappointed:

**Review by christarazi — APPROVED** [review]

File: test/k8s/updates.go (hunk 2 above)
Comment (brb) [reply test/k8s/updates.go:79]: I will keep this for now until we have resolved the external reference.

File: test/k8s/updates.go (hunk 3 above)
Comment (brb) [reply test/k8s/updates.go:77]: This is not, let me remove it :+1:

**Review by aanm — APPROVED** [review]

File: test/k8s/updates.go (hunk 2 above)
Comment (tklauser) [reply test/k8s/updates.go:79]: In principle they could still reference the manifests from a stable branch, e.g. v1.14.

**Comment by tklauser** [comment]
> /ci-runtime
> 
> Failed due to quay.io issues: https://github.com/cilium/cilium/actions/runs/7029928351/job/19128502189

Oh, this isn't a transitive failure. Looks like we need https://github.com/cilium/cilium/pull/29455 first.

**Review by tklauser — APPROVED** [review]

**Comment by aanm** [comment]
@brb should we block merging this PR until the `ci-e2e-upgrade` is more stable?

**Pull request closed without merging**